cmp/grid/Grid.js

```diff
@@ -552,6 +557,20 @@ class GridLocalModel extends HoistModel {
         };
     }
 
+    /**
+     * Force Grid to redraw rows when switching between inline and modal views
+     * {@see modalSupport}
+     */
+    modalReaction() {
+        const modalSupportModel = this.lookupModel(ModalSupportModel);
+        if (!modalSupportModel) return null;
+
+        return {
+            track: () => modalSupportModel.isModal,
+            run: () => wait().then(() => this.model.agApi.redrawRows())
```

Comment:
can also just use `delay: 0` on reaction for slightly flatter code

Reply:
Needed to do `delay: 1` (maybe MobX makes `delay: 0` a no-op?), but that worked and is cleaner